Add "Wild West"

Wesdial opened the pull request:
### Map Name
> **Wild West**

### Gamemode(s)
> **Team Deathmatch (TDM)**

### Description & Team Sizes
> This is a map that takes place in a red desert / wild west era. It can support 32 players with 16 players on each team.

### Screenshots
> [Screenshots.](https://imgur.com/a/gMcb2VV)

### XML
> [XML.](https://gist.github.com/Wesdial/0e8bb2b00e64190c70c966aa184fe9ae)

### Map Download
> [Map Download.](https://www.mediafire.com/file/6is062o9avwpxgd/Wild_West.zip/file)

### Map Checklist
> To mark as completed, replace the space between the [ ] with an `X` or click on the checkbox after submitting this pull request.
- [X] Contains a `map.xml` file with valid syntax and formatting.
- [X] The `map.xml` uses proto `1.4.0` or above (older or legacy maps are exempt, so long as the XML works).
- [X] The `map.xml` explicitly [sets the gamemode](https://pgm.dev/docs/modules/general/main#map-gamemode) (for stat-tracking).
- [X] The `map.xml` has correctly credited the map authors and contributors.
- [X] Contains `map.png` file (`290x246`).
- [X] The map was built for (or has been converted to) Minecraft 1.8.
- [X] Chunks have been pruned.
- [X] The map has been tested on a server running PGM.
- [X] I agree to submit this map under a commercial, [CC BY-SA 4.0](https://creativecommons.org/licenses/by-sa/4.0/) license.

---

### Known Issues
**No Known Issues**

BTKingly commented:
Looks pretty good. Before I accept this, wanted to point out a few things;
- The golden apples have an unbreakable tag in the starting kit (can't stack with other gapples)
- There's a few hidden chests in the map. Since you can't break blocks, are they intentional? (if so, they're empty - conversion issue?)
- 1 arrow per every 2 kills is very low. If that could be increased that'd be great

Wesdial commented:
> * The golden apples have an unbreakable tag in the starting kit

That's definietly a mistake, fixed. 

> * There's a few hidden chests in the map. Since you can't break blocks, are they intentional? (if so, they're empty - conversion issue?)

There used to be a water bucket in the chests for some reason? (mabye someone placed it there) but considering you can't place or break blocks, this seems pretty useless, so I have gone ahead and removed the chests.


> * 1 arrow per every 2 kills is very low. If that could be increased that'd be great

It was actually 2 arrows per 2 kills, but that still seems pretty low, so I went ahead and increased it to 5 arrows per 2 kills.

BTKingly commented:
Thanks! 
5 arrows per every 2 kills is still pretty low, but better. We'll have to see if it needs an increase after some playtests.
